GlideRecord/List of Child Incidents/listofchildIncident.js

```diff
@@ -0,0 +1,10 @@
+var inc = new GlideRecord('incident');
+inc.addEncodedQuery("parent_incident=9c573169c611228700193229fff72400"); // sys_id of parent Incident
```

Comment:
Sys Id should not be hardcoded in the query condition, please create a variable for it instead e.g.:
```
var parentSysId = ''; // add the sys_id of the parent incident here
...
inc.addEncodedQuery('parent_incident=' + parentSysId);
```

Reply:
Please also be consistent in using quotes vs double quotes in your code. Choose which one you want to use and stick to that for all string value definitions.